src/fetch_purchases_suppliers.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import pandas as pd
+
+
+BASE_DATA_DIR = './data/'
+
+def fetch_suppliers():
```

Comment:
This is script doesn't seem like to memory consuming, but just in case, I would write `fetch_suppliers` as a generator:

```python
def fetch_suppliers():
    # no `raw_suppliers = []`
    …
    response = requests.get(…).json()  # actually this was your `first_response`
    yield response

    …
    for page in range(2, last_page):
        …
        yield response
```

What do you think?